app/controllers/admin/navigation_links_controller.rb

```diff
@@ -8,6 +8,8 @@ class NavigationLinksController < Admin::ApplicationController
 
     def index
       @navigation_links = NavigationLink.ordered
```

Comment:
@msarit are we still using `@navigation_links` or can we remove this assignment?

Reply:
Oops! I will remove 😅